docs/src/components/APILink/index.tsx

```diff
@@ -25,8 +25,8 @@ export function APILink({ fn, children }: { fn: string; children?: React.ReactNo
 
   const docLink = useBaseUrl(`/${APIModules[module]}#${fn}`);
```

Comment:
The API link has two `latest`:

<img width="584" alt="image" src="https://github.com/user-attachments/assets/63fe54a2-0a4a-4977-ba23-53cadc39a230" />

Reply:
I also noticed this earlier this week for the preview docs, I'll land a separate PR for it.

Reply:
@harupy https://github.com/mlflow/mlflow/pull/16676